cog/api/gift.py

```
@@ -0,0 +1,64 @@
+                    "label": "前往確認",
+                    "style": 5,  # `5` 表示 Link Button
+                    "url": "https://store.scaict.org",  # 要導向的連結
+                }
+            ],
+        }
+        json_data = {
+            "embeds": [embed],
+            "components": [button],
+            "tts": False,  # Text-to-speech, 默認為 False
+        }
+        return json_data
```

Ruff

Unnecessary assignment to `json_data` before `return` statement

Remove unnecessary assignment

(RET504)